### PR TITLE
Check field values prior to running trans()

### LINKED ISSUE #3488: FormTest fails (Edge Build 435)
##### Expected behavior
Running Form Test should end in success.

##### Actual behavior

Form test fails.

```
There was 1 error:

1) FormTest::testCheckboxlistTrigger
ErrorException: htmlspecialchars() expects parameter 1 to be string, object given

/var/www/html/vendor/laravel/framework/src/Illuminate/Support/helpers.php:580
/var/www/html/modules/backend/widgets/form/partials/_field.htm:9
/var/www/html/modules/system/traits/ViewMaker.php:244
/var/www/html/modules/system/traits/ViewMaker.php:98
/var/www/html/modules/backend/widgets/form/partials/_field-container.htm:8
/var/www/html/modules/system/traits/ViewMaker.php:244
/var/www/html/modules/system/traits/ViewMaker.php:98
/var/www/html/modules/backend/widgets/form/partials/_form_fields.htm:2
/var/www/html/modules/system/traits/ViewMaker.php:244
/var/www/html/modules/system/traits/ViewMaker.php:98
/var/www/html/modules/backend/widgets/form/partials/_section.htm:17
/var/www/html/modules/system/traits/ViewMaker.php:244
/var/www/html/modules/system/traits/ViewMaker.php:98
/var/www/html/modules/backend/widgets/form/partials/_form.htm:3
/var/www/html/modules/system/traits/ViewMaker.php:244
/var/www/html/modules/system/traits/ViewMaker.php:98
/var/www/html/modules/backend/widgets/form/partials/_form-container.htm:8
/var/www/html/modules/system/traits/ViewMaker.php:244
/var/www/html/modules/system/traits/ViewMaker.php:98
/var/www/html/modules/backend/widgets/Form.php:227
/var/www/html/tests/unit/backend/widgets/FormTest.php:36

ERRORS!
Tests: 1, Assertions: 0, Errors: 1.
```

##### Reproduce steps
```
$ vendor/bin/phpunit tests/unit/backend/widgets/FormTest.php
```

##### October build
Build 435

### PR DESCRIPTION
**Pull request by petehalverson**
Fixes #3488

### CODE REVIEW

File: modules/backend/widgets/form/partials/_field.htm [hunk 1]
```diff
@@ -5,9 +5,9 @@
         <?= $this->renderFieldElement($field) ?>
 
     <?php else: ?>
-        <?php
-            $fieldComment = $field->commentHtml ? trans($field->comment) : e(trans($field->comment));
-        ?>
+        <?php if ($field->comment): ?>
```
Comment (LukeTowers) [inline modules/backend/widgets/form/partials/_field.htm]: This is merely where the value gets assigned, not where it actually gets used: https://github.com/octobercms/october/blob/master/modules/backend/widgets/form/partials/_field.htm#L18-L20

Comment (petehalverson) [reply modules/backend/widgets/form/partials/_field.htm]: Right, but it fails when running `trans($field->comment)` when `$field->comment = null`

I also considered reassigning the value in line:

```
<?php if (!$field->hidden): ?>

    <?php if (!$this->showFieldLabels($field)): ?>

        <?= $this->renderFieldElement($field) ?>

    <?php else: ?>
        <?php if ($field->label): ?>
            <label for="<?= $field->getId() ?>">
                <?= e(trans($field->label)) ?>
            </label>
        <?php endif ?>

        <?php if ($field->comment && $field->commentPosition == 'above'): ?>
            <p class="help-block before-field"><?= $field->commentHtml ? trans($field->comment) : e(trans($field->comment)) ?></p>
        <?php endif ?>

        <?= $this->renderFieldElement($field) ?>

        <?php if ($field->comment && $field->commentPosition == 'below'): ?>
            <p class="help-block"><?= $field->commentHtml ? trans($field->comment) : e(trans($field->comment)) ?></p>
        <?php endif ?>

    <?php endif ?>

<?php endif ?>
```

**Comment by petehalverson** [comment]
@jimcottrell will you check this out? I'm not sure this PR is needed.

I believe builds were failing because the packagist libraries were out of sync. Now that everything's up to date, should we still be checking those values before they hit trans()?

**Comment by LukeTowers** [comment]
Underlying issue is no longer a problem: https://github.com/octobercms/october/issues/3488

**Pull request closed without merging**